Preserved traces (#14187)
newrelic/docs-website: src/content/docs/distributed-tracing/ui-data/understand-use-distributed-tracing-ui.mdx

    * **UI display limits.** Orphaned spans may result if a trace exceeds the 10K span display limit.
  </Collapser>

  <Collapser
    id="preserved-traces"
    title="Preserved traces"
  >
    Preserved traces are similar to snapshots of original traces. They archive a full trace that has been previously viewed and has surpassed the retention period. Full traces are only available for 7 days, unless you've purchased extended retention (which would reflect automatically in the UI). However, preserved traces can exist for up to 1 year, and generally function like the original trace.

> Note that reserved traces won't display span performance data or span anomaly data. Preserved traces may also no longer be accessible if an entity involved in a preserved trace is deleted, expires, or stops reporting data.

Reviewer: ```suggestion
Note that preserved traces won't display span performance data or span anomaly data. Preserved traces may not be accessible if an entity in a preserved trace is deleted, expires, or stops reporting data.
```

    * **UI display limits.** Orphaned spans may result if a trace exceeds the 10K span display limit.
  </Collapser>

  <Collapser
    id="preserved-traces"
    title="Preserved traces"
  >

>     Preserved traces are similar to archived snapshots. They archive a trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is only available for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.

Author: ```suggestion
    Preserved traces are similar to snapshots of original traces. They archive a full trace that has been previously viewed and has surpassed the retention period. By default, a full trace is only available for 7 days, unless you've purchased extended retention (which would be reflected automatically in the UI). Preserved traces can exist for up to 1 year, and generally function like the original trace. Note that reserved traces won't display span performance data or span anomaly data. Preserved traces may also no longer be accessible if an entity involved in a preserved trace is deleted, expires, or stops reporting data.
```

>     A preserved distributed trace is like an archived snapshot of the original trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is onlyavailable for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.

Reviewer: I like framing this kind of trace as an archived snapshot, but we can unspool this first sentence for clarity. Maybe something like 'Preserved traces are similar to archived snapshots. They archive a trace that has been previously viewed and has surpassed the retention period.'

>     A preserved distributed trace is like an archived snapshot of the original trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is onlyavailable for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.

Reviewer: ```suggestion
    A preserved distributed trace is like an archived snapshot of the original trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is only available for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.
```

>     A preserved distributed trace is like an archived snapshot of the original trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is onlyavailable for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.

Author: ```suggestion
    Preserved traces are similar to archived snapshots. They archive a trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is only available for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.
```

    id="preserved-traces"
    title="Preserved traces"
  >

>     Preserved traces are similar to archived snapshots. They archive a trace that has been previously viewed and has surpassed the retention period. By default, unless you have purchased extended retention, a full trace is only available for 7 days. Preserved traces can exist for up to 1 year, and generally function like the original trace.

Reviewer: So it seems like you have to purchase something to access this feature. Once you've purchased extended retention, does this automatically appear in the UI? Do they need to enable or configure something? Might make that a little clearer.

>     However, there are some limitations to preserved traces:

Reviewer: One last thing! Can we make this preface sentence more neutral? Maybe instead of a bulleted list, just saying 'Your preserved traces won't display span performance data or span anomaly data. If an entity... etc etc'.
